src/status_im/chat/models/input.cljs

```diff
@@ -169,7 +168,7 @@
   (let [input-text    (get-in db [:chats current-chat-id :input-text])
         seq-arguments (get-in db [:chats current-chat-id :seq-arguments])
         selection     (get-in db [:chat-ui-props current-chat-id :selection])
-        chat-command  (selected-chat-command db current-chat-id)]
+        chat-command  (selected-chat-command db)]
     (current-chat-argument-position chat-command input-text selection seq-arguments)))
 
 (defn command-completion
```

Comment:
Do we have tests for this?

Reply:
Yes, we have tests for the `selected-chat-command` function